argnorm/lib.py

```diff
@@ -62,10 +64,9 @@ def map_to_aro(gene, database):
     except KeyError:
         raise Exception(f'{gene} is not in {database} database')
     else:
-        # Dealing with duplicated genes in ARO mapping table.
-        # Getting only one ARO number
         ARO = pronto.Ontology.from_obo_library('aro.obo')
-        if type(result) != str:
-            return ARO[list(set(result))[0]]
-        else:
+
+        if result in ARO:
```

Comment:
Isn't this `if`/`else` pair just the same as doing `ARO.get`?

Reply:
Wouldn't `ARO.get` also raise an error if an invalid ARO term is passed? I wanted to let the users know that though a gene is in a database that argNorm supports, argNorm does not provide an ARO mapping. I wanted to do this without raising an error, as it wouldn't be the users' fault that argNorm doesn't have a valid mapping (compared to the when the user enters an unsupported database in which case an exception is raised). 

Should I differentiate this case from the other exceptions, or would you like me to just go ahead and use `ARO.get`?

Reply:
I see the issue. Maybe just make it a bit more directed

```python
if pd.isna(result):
    return None
return ARO[result]
```